src/query/query_executor.c

```diff
@@ -23684,7 +23684,8 @@ qexec_evaluate_partition_aggregates (THREAD_ENTRY * thread_p, ACCESS_SPEC_TYPE *
 	  continue;
 	}
 
-      if (agg_ptr->function == PT_COUNT_STAR && *is_scan_needed)
+      if ((agg_ptr->function == PT_COUNT_STAR || agg_ptr->function == PT_MAX || agg_ptr->function == PT_MIN)
+	  && *is_scan_needed)
```

Comment:
This code does not exist in version 9. What is it for?

Reply:
I add the code to fix some sql fail cases, but I missed to revert it.